### ISSUE TITLE
Game crashes exactly when a Train Track is placed on the ground. UPD: Iris 1.2.6 required

### ISSUE THREAD

**Issue by tpitsunov** [Fabricators-of-Create/Create#487]
### Describe the Bug

The game crashes every time a Train Track block is rendered with iris. Other new blocks are rendered just fine as well as all the blocks from Create 0.4

the error code:
java.lang.IllegalStateException: Wrapping function stack not empty!

crash report is attached

### Reproduction Steps

described earlier

### Expected Result

well, I didn't expect a crash

### Crash Report or Log

https://pastebin.com/0MdsF0MV

### Operating System

Windows 10

### Mod Version

0.5.0c

### Minecraft Version

1.18.2

### Other Mods

all the mods are listed in the crash report

basically quite a number of small tweaks and additions and the infamous sodium+phosphor+lithium+iris+indium band

**Comment by sall0-0p** [comment]
try to remove iris, I had same problem, I removed it and everything started working

**Comment by tpitsunov** [comment]
> try to remove iris, I had same problem, I removed it and everything started working

I was actually testing various rendering mods combinations at the moment you wrote this and yeah, the problem is with iris. Thanks for the reply anyways

**Comment by tpitsunov** [comment]
So as i understand it:
a train track block cannot be rendered with iris due to some incompatibility

**Comment by TropheusJ** [comment]
Update iris

**Comment by tpitsunov** [comment]
> Update iris

it actually worked. I feel a bit stupid i didn't check for an update xD

Thanks for the reply, the issue's closed
